### PR TITLE
Voting Dashboard

### PR DESCRIPTION
**Pull request by elite0226**
## Steps to test
/projects/did:ixo:HzwjYGGEkz6F4aDPhujKJQ/detail/voting

## Screenshots
image

### CODE REVIEW

**Comment by alwyn-ixo** [comment]
@elite0226 hey - looking good!
The following questions are for me to understand the requirements that were implemented:
1. In the "ixo to Vote" screen,
1.1. Where does the 2.434M amount come from?
1.2. How do the IXO, xUSD, and IDCC tokens relate to each other?

image

**Comment by alwyn-ixo** [comment]
2. For the MyShare tab - how is the 274.00 calculated?

image

**Comment by alwyn-ixo** [comment]
3. For the My Yield tab
3.1. Where does the 587.215 come from?
3.2. How does the "0% of Funding Target" relate to "My Yield"?
3.3. How is this percentage calculated?

image

**Comment by alwyn-ixo** [comment]
4. For the My Votes tab 
4.1 Where does the 587.215 come from?
4.2 How does the "100.00% of Capital raise" get calculated?
4.3 How does "100.00% of Capital raise" relate to "My Votes"?

image

**Comment by alwyn-ixo** [comment]
The M W D views for the price chart do not work very well for the Investment Dashboard and show a flat line, instead of the transactions for the Month, Week, or Day.
Can you please check that these ones work correctly?

image

**Comment by elite0226** [comment]
updates on Price card.
image

**Comment by elite0226** [comment]
linked `bondDid`: https://testnet.ixo.earth/rest/bonds/did:ixo:NxcsZ74HE7UqSfxXz6p14b

**Comment by elite0226** [comment]
> 1. In the "ixo to Vote" screen,
> 1.1. Where does the 2.434M amount come from?
> 1.2. How do the IXO, xUSD, and IDCC tokens relate to each other?

1.1. 
image
1.2. 
This was my mistake. there should be `xUSD` for this bond. should be replaced with `IXO`

**Comment by elite0226** [comment]
> 2. For the MyShare tab - how is the 274.00 calculated?

2.
image

**Comment by elite0226** [comment]
> 3. For the My Yield tab
> 3.1. Where does the 587.215 come from?
> 3.2. How does the "0% of Funding Target" relate to "My Yield"?
> 3.3. How is this percentage calculated?

3.1.
image
3.2.
3.3.
`0` is coming from the `goal` of linked claims. Not sure. 

Need to confirm with @ig-shaun

**Comment by elite0226** [comment]
> 4. For the My Votes tab
> 4.1 Where does the 587.215 come from?
> 4.2 How does the "100.00% of Capital raise" get calculated?
> 4.3 How does "100.00% of Capital raise" relate to "My Votes"?

4.1. 
image
4.2. 
(`available_reserve` / `current_reserve`) * 100
i.e. (587214931 / 587214931) * 100 = **100.00%**
4.3. 
Not sure to be honest. UI component has been forked from bonds dashboard.
Need to confirm with @ig-shaun to display the correct contexts

**Comment by demondayza** [comment]
Thanks @elite0226 , will discuss with @alwyn-ixo

**Comment by elite0226** [comment]
> The M W D views for the price chart do not work very well for the Investment Dashboard and show a flat line, instead of the transactions for the Month, Week, or Day.

updated the chart behavior when filtered in M, D, W
image
image

File: src/modules/BondModules/bond/bond.actions.ts [hunk 1]
```diff
@@ -26,6 +27,13 @@ import moment from 'moment'
 // const NEW_BLOCKSYNC_API = 'http://136.244.115.236:8080'
```
Comment (demondayza) [inline src/modules/BondModules/bond/bond.actions.ts:27]: Please remove

**Pull request merged**